src/main/java/jp/ikedam/jenkins/plugins/scoringloadbalancer/rules/BuildResultScoringRule.java

```diff
@@ -137,31 +145,72 @@ public BuildResultScoringRule(
     @Override
     public boolean updateScores(Task task, WorkChunk wc, Mapping m, NodesScore nodesScore) {
         for (SubTask subtask : wc) {
-            if (!(subtask instanceof AbstractProject)) {
-                return true;
+
+            AbstractBuild<?, ?> freestylebuild = null;
+            WorkflowRun pipelinebuild = null;
+
+            if ((subtask instanceof AbstractProject)) {
+                AbstractProject<?, ?> project = (AbstractProject<?, ?>) subtask;
```

Comment:
```suggestion
                var project = (AbstractProject<?, ?>) subtask;
```

Use `var` where possible